migtests/tests/pg/partitions/snapshot.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
```

Comment:
Assuming that the ONLY change here is that you're specifying ROW_COUNT and essentially making generate_series dynamic.

Reply:
Yes correct